mypy/nodes.py

```diff
@@ -1917,301 +1931,6 @@ def accept(self, visitor: ExpressionVisitor[T]) -> T:
-    Where the class is used in an actual type, it's in the form of an
-    Instance, which amounts to a type application of the tycon to
-    the appropriate number of arguments.
-    """
-
-    _fullname = None  # type: str          # Fully qualified name
-    # Fully qualified name for the module this type was defined in. This
-    # information is also in the fullname, but is harder to extract in the
-    # case of nested class definitions.
-    module_name = None  # type: str
-    defn = None  # type: ClassDef          # Corresponding ClassDef
-    # Method Resolution Order: the order of looking up attributes. The first
-    # value always to refers to this class.
-    mro = None  # type: List[TypeInfo]
-
-    declared_metaclass = None  # type: Optional[mypy.types.Instance]
-    metaclass_type = None  # type: mypy.types.Instance
-
-    subtypes = None  # type: Set[TypeInfo] # Direct subclasses encountered so far
-    names = None  # type: SymbolTable      # Names defined directly in this type
-    is_abstract = False                    # Does the class have any abstract attributes?
-    abstract_attributes = None  # type: List[str]
-    # Classes inheriting from Enum shadow their true members with a __getattr__, so we
-    # have to treat them as a special case.
-    is_enum = False
-    # If true, any unknown attributes should have type 'Any' instead
-    # of generating a type error.  This would be true if there is a
-    # base class with type 'Any', but other use cases may be
-    # possible. This is similar to having __getattr__ that returns Any
-    # (and __setattr__), but without the __getattr__ method.
-    fallback_to_any = False
-
-    # Information related to type annotations.
-
-    # Generic type variable names
-    type_vars = None  # type: List[str]
-
-    # Direct base classes.
-    bases = None  # type: List[mypy.types.Instance]
-
-    # Another type which this type will be treated as a subtype of,
-    # even though it's not a subclass in Python.  The non-standard
-    # `@_promote` decorator introduces this, and there are also
-    # several builtin examples, in particular `int` -> `float`.
-    _promote = None  # type: mypy.types.Type
-
-    # Representation of a Tuple[...] base class, if the class has any
-    # (e.g., for named tuples). If this is not None, the actual Type
-    # object used for this class is not an Instance but a TupleType;
-    # the corresponding Instance is set as the fallback type of the
-    # tuple type.
-    tuple_type = None  # type: Optional[mypy.types.TupleType]
-
-    # Is this a named tuple type?
-    is_named_tuple = False
-
-    # If this class is defined by the TypedDict type constructor,
-    # then this is not None.
-    typeddict_type = None  # type: Optional[mypy.types.TypedDictType]
-
-    # Is this a newtype type?
-    is_newtype = False
-
-    # Alternative to fullname() for 'anonymous' classes.
-    alt_fullname = None  # type: Optional[str]
```

Comment:
This field is removed. It is never set to anything other than `None`. If this PR is rejected, I can post a separate one just for this removal.